components/twitter/actions/get-user/get-user.mjs

```diff
@@ -0,0 +1,42 @@
+import twitter from "../../twitter.app.mjs";
+
+export default {
+  key: "twitter-get-user",
+  name: "Get User",
+  description: "Return information about the user specified by ID or screen name parameter",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    twitter,
+    userId: {
+      propDefinition: [
+        twitter,
+        "userId",
+      ],
+      optional: true,
+    },
+    screenName: {
+      propDefinition: [
+        twitter,
+        "screenName",
+      ],
+      optional: true,
+    },
+  },
+  async run() {
+    const {
+      userId,
+      screenName,
+    } = this;
+
+    if (!userId && !screenName) {
+      throw new Error("This action requires either User ID or Screen Name. Please enter one or the other above.");
+    }
+
+    return await this.twitter.lookupUsers([
```

Comment:
Since this is an async function. [You don't need to `return await`](https://eslint.org/docs/rules/no-return-await). Could you please remove it?